packages/react-composites/src/composites/CallWithChatComposite/styles/CallWithChatCompositeStyles.ts

```diff
@@ -15,7 +15,42 @@ export const compositeOuterContainerStyles: IStackStyles = {
     // of the height of the child items.
     // If this css property is to be removed, test the participant pane correctly uses the scroll overflow when there a lot
     // of participants (i.e. beyond screen height) and where one participant is sharing their screen.
-    flexWrap: 'wrap'
+    flexWrap: 'wrap',
+
+    // Create a new stacking context so that DrawerMenu can be positioned absolutely.
+    position: 'relative'
+  }
+};
+
+/** @private */
+export const callCompositeContainerStyles: IStackStyles = {
+  root: {
+    // Start a new stacking context so that any `position:absolute` elements
+    // inside the call composite do not compete with its siblings.
+    position: 'relative'
+  }
+};
+
+/** @private */
+export const controlBarContainerStyles: IStackStyles = {
+  root: {
+    // Start a new stacking context so that any `position:absolute` elements
+    // inside the control bar do not compete with its siblings.
+    position: 'relative'
+  }
+};
+
+/** @private */
+export const drawerContainerStyles: IStackStyles = {
+  root: {
+    position: 'absolute',
+    top: 0,
+    left: 0,
+    width: '100%',
+    height: '100%',
+    // Any zIndex > 0 will work because this is the only absolutely
```

Comment:
what happens if in future another absolutely positioned element is introduced to the UI?
Do we have a well defined rule for this drawer. If it's always going to be on top when triggered, might I suggest using something like 99

Reply:
We don't have strict rules here. I tend to stick to the lowest possible numbers. (So if I need one more explicitly ordered item, either it will have zIndex 2, or this one will be bumped to 2 (and that will have 1).

Using something like 99 might encourage using 55 when somebody wants something in the "middle".

Best just stick to lowest numbers and force us to think about the order.